artview/components/menu.py

```diff
@@ -361,7 +361,7 @@ def _openfile(self):
      
         # Read the data from file
         try:
-            radar = pyart.io.read(self.filename)
+            radar = pyart.io.read(self.filename, delay_field_loading=True)
```

Comment:
definitely this is a PyART bug, `pyart.io.read` is just getting **kwargs and passing it to the other read functions, no test for `delay_field_loading` is made. I will work with @jjhelmus to correct this.